tests/unit/libbpf_test.cpp

```diff
@@ -402,23 +431,77 @@ TEST_CASE("libbpf map", "[libbpf]")
     REQUIRE(result < 0);
     REQUIRE(errno == EINVAL);
 
+    // Wrong fd type
```

Comment:
nit: add period to end of 434, 439, etc. for consistency with line 444, 459, 500, etc.